Datasources: fixed long error message overflowing container

thisisobate opened the pull request:
Signed-off-by: Uchechukwu Obasi <obasiuche62@gmail.com>

<!--

Thank you for sending a pull request! Here are some tips:

1. If this is your first time, please read our contribution guide at https://github.com/grafana/grafana/blob/master/CONTRIBUTING.md

2. Ensure you include and run the appropriate tests as part of your Pull Request.

3. In a new feature or configuration option, an update to the documentation is necessary. Everything related to the documentation is under the docs folder in the root of the repository.

4. If the Pull Request is a work in progress, make use of GitHub's "Draft PR" feature and mark it as such.

5. If you can not merge your Pull Request due to a merge conflict, Rebase it. This gets it in sync with the master branch.

6. Name your PR as "<FeatureArea>: Describe your change", e.g. Alerting: Prevent race condition. If it's a fix or feature relevant for the changelog describe the user impact in the title. The PR title is used to auto-generate the changelog for issues marked with the "add to changelog" label.

-->

**What this PR does / why we need it**:
This PR fixes long error messages overflowing its container. 
**Which issue(s) this PR fixes**:

<!--

- Automatically closes linked issue when the Pull Request is merged.

Usage: "Fixes #<issue number>", or "Fixes (paste link of issue)"

-->

Fixes #29196 

**Special notes for your reviewer**:
Long error messages now fit into its parent container
<img width="1792" alt="Screenshot 2020-11-27 at 17 00 55" src="https://user-images.githubusercontent.com/29557702/100470754-21eabb00-30d9-11eb-8302-2997f925a32d.png">

Commit:
Fixed: error message overflow container

Signed-off-by: Uchechukwu Obasi <obasiuche62@gmail.com>

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/grafana/grafana?pullRequest=29440) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/grafana/grafana?pullRequest=29440) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/grafana/grafana?pullRequest=29440) it.</sub>

torkelo commented on public/sass/components/_alerts.scss:
```diff
@@ -78,6 +79,7 @@
 
 .alert-body {
   flex-grow: 1;
+  width: 50%;
```
why is the alert body 50%?

thisisobate replied:
Creating a child container- in this case, `Alert-body` -without explicitly setting its width causes the content to overflow.
Because the data is dynamically generated, it's best to set constraints so as to avoid issues like this.

thisisobate replied:
I replaced `width` with `min-width` which seems to work pretty fine.
wdyt? @torkelo

thisisobate committed:
used min-width property for better styling

Signed-off-by: Uchechukwu Obasi <obasiuche62@gmail.com>

torkelo committed:
Switched to Alert component

torkelo reviewed (commented):
I updated the page to use our new Alert component and fixed the issue in that one as well 

seems like with 

```css
overflow-wrap: break-word; // (for chrome and safari)
word-break: break-word; // (for firefox)
```

That seems to work for me without any min-width

thisisobate commented:
> I updated the page to use our new Alert component and fixed the issue in that one as well
> 
> seems like with
> 
> ```css
> overflow-wrap: break-word; // (for chrome and safari)
> word-break: break-word; // (for firefox)
> ```
> 
> That seems to work for me without any min-width

Wow...that's quite different for me.
I just removed `min-width: 0` and the error is still persistent
<img width="1792" alt="Screenshot 2020-12-01 at 13 56 53" src="https://user-images.githubusercontent.com/29557702/100743967-5a421000-33dd-11eb-9010-b14df8df571f.png">

torkelo commented:
Did you add

```
overflow-wrap: break-word; 
word-break: break-word;
```
to the body?

thisisobate commented:
yes, I did.
<img width="1792" alt="Screenshot 2020-12-01 at 14 02 25" src="https://user-images.githubusercontent.com/29557702/100744298-e18f8380-33dd-11eb-91cc-e259516a8206.png">

thisisobate commented:
@torkelo If you've got some free time, maybe we could jump on a quick pair programming session.

torkelo committed:
Fixed passing aria-label to Alert component

torkelo commented:
@thisisobate be sure to pull latest from this branch, have pushed two changes. I cannot replicate the issue anymore in chrome or firefox

thisisobate commented:
Alright.
Thanks!

torkelo committed:
Fixed e2e test (I hope)

torkelo committed:
another attempt to fix e2e

torkelo committed:
Fixed display name
